components/clp-py-utils/clp_py_utils/profiling_utils.py

```
@@ -0,0 +1,257 @@
+    """
+    try:
+        # Get the session for logging
+        session = profiler.last_session
+        if not session:
+            logger.debug(f"No profiling session for {section_name}")
+            return
+
+        duration = session.duration
+        sample_count = session.sample_count
+
+        timestamp = datetime.datetime.now().strftime("%Y%m%d_%H%M%S_%f")
```

Ruff (0.14.1)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)